draft-normington-jsonpath-latest.txt

```diff
@@ -205,9 +205,14 @@ Internet-Draft   JavaScript Object Notation (JSON) Path        July 2020
    node: the input document.
 
    A matcher may match a node and, if it matches, may then select zero
-   or more descendants of the node for further processing.  But there's
-   more to it than this: each matcher acts on a list of nodes and
-   produces a list of nodes, as follows.
+   or more descendants of the node for further processing.  A
+   syntactically valid matcher MUST NOT produce errors and so some
+   operations which might be considered erroneous, such as indexing
+   beyond the end of an array, simply result in fewer descendants being
+   selected.
```

Comment:
I suggest breaking this into two sentences. "A syntactically valid matcher MUST NOT produce errors. This means that some operations which might be considered erroneous, such as indexing beyond the end of an array, result in fewer descendants being selected".

Reply:
Yes, I like that thanks.